website/content/en/preview/concepts/_index.md

```diff
@@ -116,7 +117,18 @@ To perform these actions, Karpenter simulates all pods being evicted from a cand
 
 If as a result of the scheduling simulation all pods can run on existing nodes, the candidate node is simply deleted.  If all pods can run on a combination of existing nodes and a cheaper node, we launch the cheaper node and delete the candidate node which causes the pods to be evicted and re-created by their controllers in order to be rescheduled.
 
-For Node Replacement to work well, your provisioner must allow selecting from a variety of instance types with varying amounts of allocatable resources.  Consolidation will only consider launching nodes using instance types which are allowed by your provisioner. 
+For Node Replacement to work well, your provisioner must allow selecting from a variety of instance types with varying amounts of allocatable resources.  Consolidation will only consider launching nodes using instance types which are allowed by your provisioner.
+
+### Interruption
+
+If interruption-handling is enabled for the controller, Karpenter will watch for upcoming involuntary interruption events that would cause disruption to your workloads. These interruption events include:
+
+* Spot Interruption Warnings
+* Scheduled Change Health Events (Maintenance Events)
+* Instance Terminating Events
+* Instance Stopping Events
+
+When Karpenter detects one of these events will occur to your nodes, it automatically cordons, drains, and terminates the node(s) ahead of the interruption event to give the maximum amount of time for workload cleanup prior to compute disruption. This enables scenarios where the `terminationGracePeriod` for your workloads may be long or cleanup for your workloads is critical, and you want enough time to be able to gracefully clean-up your pods.
```

Comment:
can you make a note that we don't support Rebalance Recommendations today so users will need to continue to use NTH if they want that functionality?